src/Router.js

```diff
@@ -1,223 +0,0 @@
-// 3rd party libraries - core
```

Comment:
Hmm I guess I already deleted the file, lol. Anyway, I'll probably add `Router.js` back to hold our `react-navigation` logic, later.